server/src/main/java/org/elasticsearch/cluster/metadata/RepositoriesMetadata.java

```diff
@@ -45,6 +47,12 @@
 
     public static final String TYPE = "repositories";
 
+    /**
+     * Serialization parameter used to hide the {@link RepositoryMetadata#generation()} and {@link RepositoryMetadata#pendingGeneration()}
+     * in {@link org.elasticsearch.action.admin.cluster.repositories.get.GetRepositoriesResponse}.
+     */
+    public static final String HIDE_GENERATIONS_PARAM = "hide_generations";
```

Comment:
I think we could remove this, given that we don't have the `GetRepositoriesResponse`